UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/HealthCheck/RNWDatabaseHealthCheck.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using UKHO.MaritimeSafetyInformation.Common.Logging;
+
+namespace UKHO.MaritimeSafetyInformation.Common.HealthCheck
+{
+    public class RNWDatabaseHealthCheck : IHealthCheck
+    {
+        private readonly IRNWDatabaseHealthClient _rnwDatabaseHealthClient;
+        private readonly ILogger<RNWDatabaseHealthCheck> _logger;
+
+        public RNWDatabaseHealthCheck(IRNWDatabaseHealthClient rnwDatabaseHealthClient, ILogger<RNWDatabaseHealthCheck> logger)
+        {
+            _rnwDatabaseHealthClient = rnwDatabaseHealthClient;
+            _logger = logger;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            HealthCheckResult healthCheckResult = await _rnwDatabaseHealthClient.CheckHealthAsync();
```

Comment:
Pass cancellationToken in all subsequent calls.
```suggestion
            HealthCheckResult healthCheckResult = await _rnwDatabaseHealthClient.CheckHealthAsync(cancellationToken);
```

Reply:
Changes done.